norduniclient/models.py

```diff
@@ -277,6 +277,25 @@ def get_ports(self):
             """
         return core.query_to_list(self.manager, q, handle_id=self.handle_id)
 
+    def add_property(self, property, value):
```

Comment:
Should not add_property and remove_property be part of the BaseNodeModel?

Also you can also change the node.data and call set_node_properties from core.py with the full set of properties when you want to save the changes. This would result in fewer db calls but might not be necessary anymore.

But I am not against adding add_property and remove_property.

Reply:
The methods are now on BaseNodeModel. Later I saw how to add properties using the data attribute, but those methods seemed handy anyways, I thought that CommonQueries was the class for those methods, but BaseNodeModel fits better.